src/storage/v2/durability/metadata.hpp

```diff
@@ -41,6 +41,7 @@ struct RecoveredIndicesAndConstraints {
   struct IndicesMetadata {
     std::vector<LabelId> label;
     std::vector<std::pair<LabelId, PropertyId>> label_property;
+    std::vector<std::pair<LabelId, PropertyId>> point_label_property;
```

Comment:
Currently you can do that but it will be disallowed with the introduction of type constraints (coming in 2.20). Spatial indices will construct a type Point constraint on the label-property combination.